public/index.html

```diff
@@ -1,11 +1,21 @@
 <!doctype html>
 <head>
   <title>o2znz</title>
+  <link rel="stylesheet" type="text/css" href="./styles.css" />
 </head>
 <body>
   <div id="photo-gallery"></div>
+  <div id="checkout"></div>
+  <div id="reviews"></div>
+  <div id="recommendations"></div>
+  <div id="experiences"></div>
   <script src="https://unpkg.com/react@16/umd/react.development.js" crossorigin></script>
   <script src="https://unpkg.com/react-dom@16/umd/react-dom.development.js" crossorigin></script>
+  <script src="//unpkg.com/styled-components/dist/styled-components.min.js"></script>
   <script src="http://localhost:3001/:listingid/bundle.js"></script>
+  <script src="http://localhost:3002/bundle.js"></script>
+  <script src="http://localhost:3003/bundle.js"></script>
+  <script src="http://localhost:3004/bundle.js"></script>
+  <script src="http://localhost:3005/:listingid/bundle.js"></script>
```

Comment:
for some reason I could only get this to work with :listingid (on proxy side and module side), we should probably play around with it a bit more @BrewBeast

Reply:
looks like this self-resolved somehow!